Tasks/Tesevich/Task9/back.py

```diff
@@ -0,0 +1,72 @@
+import json
+import uuid
+class Films:
+    def __init__(self, film, autor, year, genre):
+        self.__all_in = []
+        self.__id = str(uuid.uuid4())
+        self.__film = film
+        self.__autor = autor
+        self.__year = year
+        self.__genre = genre
```

Comment:
it's not very good implementation of oop, mind you

would be better to have a separate class for the Film data and another one for the "FilmManager" entity, the Film should only store data about a singular film, the manager should perform all other operations